client/overview/task-list/tasks/po-task.tsx

```diff
@@ -117,20 +120,19 @@ export const getVerifyBankAccountTask = (): any => {
 
 	return {
 		key: 'verify-bank-details-po',
-		level: level,
 		title: title,
 		content: description,
+		level: level,
 		completed: false,
+		expanded: true,
+		isDismissable: false,
+		showActionButton: true,
+		action: handleClick,
 		onClick: () => {
-			window.location.href = addQueryArgs( wcpaySettings.connectUrl, {
```

Comment:
you have to leave onClick handler otherwise it's not clickable in Woo Home, same as reconnect wp com

Reply:
Fixed in https://github.com/Automattic/woocommerce-payments/commit/5e3cfaff364d05b2ee0e989a4b435b7bc19deb79. 🙂